src/tests/test_asyncio.py

```diff
@@ -412,9 +412,9 @@ def test_await_pyproxy_eval_async(selenium):
             from js import fetch
             await (await fetch('packages.json')).json()
         `);
-        let packages = await c;
+        let result = await c;
```

Comment:
We should move to fetching some other test JSON data by default in tests. I don't know how often this will happen but I think it's not ideal to have a bunch of tests asserting stuff about the structure of `packages.json`.